easybuild/easyconfigs/p/Python/Python-2.7.8-goolf-1.5.14.eb

```diff
@@ -0,0 +1,160 @@
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('bzip2', '1.0.6'),          # for bz2 package
+#    ('zlib', '1.2.8'),          # use version selected by libxslt below
+    ('libreadline', '6.2'),
+    ('ncurses', '5.9'),
+    ('LLVM', '3.2'),             # for Numba
+    ('SQLite', '3.8.4.3'),       # for IPython and general use
+    ('libxslt', '1.1.28'),       # for lxml.  brings in libxml2 and zlib dependencies.
+    ('CUDA', '6.0.37','', True), # for pycuda, duh.
+#   ('OpenSSL', '1.0.1f'),       # OS dependency should be preferred for security reasons
+
+    ('libjpeg-turbo','1.3.0'),   # For matplotlib and Pillow
+    ('libpng','1.6.10'),         # For matplotlib and Pillow
+    ('freetype', '2.5.3'),       # For matplotlib and Pillow
+
+]
+
+# order is important!
+exts_list = [
+    ('setuptools', '5.7', {
+        'source_urls': ['http://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('ipython', '2.2.0', {
+        'modulename': 'IPython',
+        'source_urls': ['http://pypi.python.org/packages/source/i/ipython/'],
+    }),
+    ('pip', '1.5.6', {
+        'source_urls': ['http://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('virtualenv', '1.11.6', {
+        'source_urls': ['http://pypi.python.org/packages/source/v/virtualenv/'],
+    }),
+    ('nose', '1.3.3', {
+        'source_urls': ['http://pypi.python.org/packages/source/n/nose/'],
+    }),
+    ('line_profiler', '1.0b3', {
+        'source_urls': ['http://pypi.python.org/packages/source/l/line_profiler/'],
+    }),
+    ('numpy', numpyversion, {
+        'source_urls': [('http://sourceforge.net/projects/numpy/files/NumPy/%s' % numpyversion, 'download')],
+        'patches': [
+            'numpy-%s_distutils_multiple-lib-dirs.patch' % ('1.8.1' if numpyversion.startswith('1.8') else numpyversion),
+        ],
+    }),
+    ('scipy', scipyversion, {
+        'source_urls': [('http://sourceforge.net/projects/scipy/files/scipy/%s' % scipyversion, 'download')],
+    }),
+    ('Cython', '0.20.2', {
+        'source_urls': ['http://www.cython.org/release/'],
+    }),
+
+
+    ('networkx', '1.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx/'],
+        'modulename': 'Bio',
+    }),
+    ('biopython', '1.64', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/biopython/'],
+        'modulename': 'Bio',
+    }),
+
+    ('Pillow', '2.5.3', {
+        'source_tmpl': SOURCE_ZIP, #'%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/P/Pillow/'],
+        'modulename': 'PIL',
+    }),
+    ('matplotlib', '1.3.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
+    }),
+    ('skimage', '0.10.1', {
+        'source_tmpl': 'scikit-image-%(version)s.tar.gz',
+        'source_urls': ['http://pypi.python.org/packages/source/s/scikit-image/'],
+    }),
+
+    ('lxml', '3.3.5', {
```

Comment:
adding extra extensions is fine, as long as they don't introduce extra dependencies, but we should try and keep the different Python easyconfigs (for the same version of Python) in sync...